lib/kuniri/language/ruby/function_behavior_ruby.rb

```diff
@@ -64,15 +64,18 @@ def handling_default_parameter(pLine)
 
       # Override
       def remove_unnecessary_information(pLine)
-        return pLine.gsub(/\s+|\(|\)/, '') if pLine =~ /\s+|\(|\)/
-        return pLine
+        return pLine if pLine !~ /\s+|\(|\)/
+
+        nLine = pLine.gsub(/\s+|\(|\)/, '')
+
+        nLine != '' ? nLine : nil
       end
 
       # Override
       def handling_parameter(pLine)
         # Handling with parenthesis and without it.
-        if pLine =~ /\(.+\)/
-          partial = get_parameters(pLine, /\(.+\)/)
+        if pLine =~ /\(.*\)/
```

Comment:
Why change '+' by '*'? Usually, more restrict is better for regex. Can you just explain this point?